### ISSUE TITLE
error redirect about:blank when check website on https://pagespeed.web.dev/

### ISSUE THREAD

**Issue by superantim** [theajack/disable-devtool#53]
i think the bot google can't not crawl data when website using script
i set default setting disable-devtool-auto
this really bad for seo
![image](https://user-images.githubusercontent.com/45330570/219108411-dd3a731c-c693-4c78-bc49-c5c7499aa8d6.png)
![image](https://user-images.githubusercontent.com/45330570/219108666-56088e28-ee2f-4c29-8111-64e1692b8313.png)

**Comment by theajack** [comment]
You can try the npm way @superantim

**Comment by superantim** [comment]
> You can try the npm way @superantim 

it work?
you mean using like this
<script disable-devtool-auto src='https://cdn.jsdelivr.net/npm/disable-devtool@latest'></script>

**Comment by theajack** [comment]
@superantim 

No

First, I tried chrome analysis, and the performance of seo related data is normal.  I think there should be no relationship between SEO and script usage.

![image](https://user-images.githubusercontent.com/23377279/219540767-4fde92be-daf6-4311-a836-138a14df615e.png)

If you have problems on your side, you can use npm to install and use, like this：

```
npm i disable-devtool
```

```js
import DisableDevtool from 'disable-devtool';
DisableDevtool();
```

**Comment by theajack** [comment]
@superantim 
I misunderstood you at first
It may be that the disabled function also blocked the google robot
I'll see if there's a way to ignore google's robots

**Comment by superantim** [comment]
> @superantim 
> 
> I misunderstood you at first
> 
> It may be that the disabled function also blocked the google robot
> 
> I'll see if there's a way to ignore google's robots

using the ?ddtk tag everything works fine because the script has been disabled, if there is a way to recognize bots of search engines that would be great

**Comment by theajack** [comment]
This issue was fixed in v0.3.3 by identifying and ignoring Lighthouse and some common search engine bots, Thanks for your help

@superantim

**Comment by superantim** [comment]
> error again in version 0.3.4, pls fix ![image](https://user-images.githubusercontent.com/45330570/234310005-2cb094e8-232c-4b2a-95d3-de3d784f0032.png)

@theajack pls fix this
